src/main.rs

```diff
@@ -75,3 +80,11 @@ fn download_tar(target: &str) -> Result<TarArchive, CliError> {
     Ok(tar::Archive::new(decoded_response))
 }
 
+
+fn store_token_in_home(token: String) -> std::io::Result<()>{
+    let home = env::var("HOME").unwrap();
+    let config_path = format!("{}/.sharpener-config", home);
+    let mut file = File::create(config_path)?;
+    file.write_all(token.as_bytes())?;
```

Comment:
This may be a future thing but I would store the config as a json instead of a plain string. This gives you flexibility in the future to add new fields.